shroud/visitor.py

```diff
@@ -0,0 +1,79 @@
+# The MIT License (MIT)
```

Comment:
Added.  Is there any issues with adding other code?  MIT license is pretty liberal.

Reply:
There shouldn't be an issue, but it is good form to list (and acknowledge) external code.  It can also be helpful down the road to remind users where the code came from.